activiti-spring-identity/src/test/java/org/activiti/spring/identity/IdentityApplication.java

```diff
@@ -20,20 +19,19 @@ public static void main(String[] args) {
     }
 
     @Bean
-    public UserDetailsService myUserDetailsService() {
-        InMemoryUserDetailsManager inMemoryUserDetailsManager = new InMemoryUserDetailsManager();
+    public UserDetails myUserDetailsService() {
```

Comment:
@pancudaniel7 you changed this again?? It needs to be UserDetailsService which is a spring class.